src/assisted_test_infra/download_logs/download_logs.py

```
@@ -54,6 +54,67 @@
+        if isinstance(ca_list, list) and ca_list:
+            if not (len(ca_list) == 1 and ca_list[0].get("source") == "censored"):
+                data["ignition"]["security"]["tls"]["certificateAuthorities"] = [
+                    {"source": "censored", "verification": {}}
+                ]
+                log.debug("Censored certificate authorities")
+                return True
+    except KeyError:
+        pass
+    return False
+
+    if changes_made:
```

Ruff (0.11.9)

Undefined name `changes_made`

(F821)